### PR TITLE
Feature/config limit read since

### PR DESCRIPTION
**Pull request by rgarver**
Let the batch size for Mysql#read_since be configured and passed along to Mysql#read

### CODE REVIEW

**Comment by evantahler** [comment]
again, looks good!

**Comment by evantahler** [comment]
let me know when your current batch of updates is done, and I'll cut another version.

**Comment by rgarver** [comment]
Hey @evantahler I pulled a little more in to this PR than I originally intended. This is definitely worth a major version bump since there are some breaking API changes around the Mysql patterns.

File: README.md [hunk 1]
```diff
@@ -1,16 +1,9 @@
 # Forklift ETL
 
----
-
-## THIS TOOL IS NO LONGER SUPORTED.
-It has been succeded by [Empujar](https://github.com/taskrabbit/empujar)
```
Comment (evantahler) [inline README.md]: If you are removing this method, that assumes you want to support it @rgarver... is that the case?!

**Comment by rgarver** [comment]
We are actively using it and we've been hacking it locally. About time to give back

**Pull request merged**